src/Domain/Amr/Flag.cpp

```diff
@@ -6,7 +6,7 @@
 
 #include <ostream>
 
-namespace amr {
+namespace domain::amr {
```

Comment:
What do you think about moving the `Amr` directory to `ParallelAlgorithms/Amr` and keeping the `amr` namespace top-level? I think things with actions should be in `ParallelAlgorithms/`, `Evolution/` or `Elliptic/`.

Reply:
I have no strong opinion either way on the location of the directory, or whether or not the `amr` namespace is inside of the `domain` namespace; I just did what seemed most natural.  I would, however, like the decision made before I add more code 🙂 .